.idea/modules.xml

```diff
@@ -3,12 +3,13 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/buildSrc/buildSrc.iml" filepath="$PROJECT_DIR$/buildSrc/buildSrc.iml" />
-      <module fileurl="file://$PROJECT_DIR$/componentapiexample/componentapiexample.iml" filepath="$PROJECT_DIR$/componentapiexample/componentapiexample.iml" />
-      <module fileurl="file://$PROJECT_DIR$/gini-vision-lib-android.iml" filepath="$PROJECT_DIR$/gini-vision-lib-android.iml" />
-      <module fileurl="file://$PROJECT_DIR$/ginivision/ginivision.iml" filepath="$PROJECT_DIR$/ginivision/ginivision.iml" />
-      <module fileurl="file://$PROJECT_DIR$/ginivision-accounting-network/ginivision-accounting-network.iml" filepath="$PROJECT_DIR$/ginivision-accounting-network/ginivision-accounting-network.iml" />
-      <module fileurl="file://$PROJECT_DIR$/ginivision-network/ginivision-network.iml" filepath="$PROJECT_DIR$/ginivision-network/ginivision-network.iml" />
-      <module fileurl="file://$PROJECT_DIR$/screenapiexample/screenapiexample.iml" filepath="$PROJECT_DIR$/screenapiexample/screenapiexample.iml" />
+      <module fileurl="file://$PROJECT_DIR$/componentapiexample/componentapiexample.iml" filepath="$PROJECT_DIR$/componentapiexample/componentapiexample.iml" group="gini-vision-lib-android/componentapiexample" />
+      <module fileurl="file://$PROJECT_DIR$/gini-vision-lib-android.iml" filepath="$PROJECT_DIR$/gini-vision-lib-android.iml" group="gini-vision-lib-android" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/modules/gini-vision-lib-android_buildSrc.iml" filepath="$PROJECT_DIR$/.idea/modules/gini-vision-lib-android_buildSrc.iml" group="gini-vision-lib-android/gini-vision-lib-android_buildSrc" />
+      <module fileurl="file://$PROJECT_DIR$/ginivision/ginivision.iml" filepath="$PROJECT_DIR$/ginivision/ginivision.iml" group="gini-vision-lib-android/ginivision" />
+      <module fileurl="file://$PROJECT_DIR$/ginivision-accounting-network/ginivision-accounting-network.iml" filepath="$PROJECT_DIR$/ginivision-accounting-network/ginivision-accounting-network.iml" group="gini-vision-lib-android/ginivision-accounting-network" />
+      <module fileurl="file://$PROJECT_DIR$/ginivision-network/ginivision-network.iml" filepath="$PROJECT_DIR$/ginivision-network/ginivision-network.iml" group="gini-vision-lib-android/ginivision-network" />
+      <module fileurl="file://$PROJECT_DIR$/screenapiexample/screenapiexample.iml" filepath="$PROJECT_DIR$/screenapiexample/screenapiexample.iml" group="gini-vision-lib-android/screenapiexample" />
     </modules>
   </component>
 </project>
```

Comment:
See above

Reply:
The latest Android Studio version changed this. I added it because the file was already tracked and otherwise the changes would always be recreated by Android Studio and would pollute `git status`.